docs/source/platform/schema-validation.md

```diff
@@ -147,7 +146,31 @@ After analyzing the changes against current usage metrics, Apollo will identify
 2. **Warning**: There are potential problems that may come from this change, but no clients are immediately impacted.
 3. **Notice**: This change is safe and will not break current clients.
 
-The more [performance metrics](./performance.html) that Apollo has, the better the report of these changes will become.
+<h3 id="cli-advanced">Advanced CLI Usage</h3>
```

Comment:
I think that many people reading this will have other features that they'd like to have (e.g. ignoring errors in deprecated fields, ignoring a specific "whitelist" of field changes, modifying the "to" parameter, etc.) and I think it'd be wise to tell them "we know this isn't everything you might want to tune. If you have a specific request, we'd love to hear it. Please open up an issue <here> and provide as much detail as you can"

Reply:
Resolved in edb9ee34211ebc1224c4b58e126bc1c5674f780b